src/main/java/co/inventorsoft/jdbc/controller/MailController.java

```diff
@@ -0,0 +1,71 @@
+package co.inventorsoft.jdbc.controller;
+
+import co.inventorsoft.jdbc.model.Mail;
+import co.inventorsoft.jdbc.service.MailService;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+
+
+@RestController
```

Comment:
When you work with template engine (Thymeleaf), your controllers are not `@RestController`s